mmv1/third_party/terraform/tests/resource_compute_instance_test.go.erb

```diff
@@ -753,6 +753,30 @@ func TestAccComputeInstance_scratchDisk(t *testing.T) {
 	})
 }
 
+func TestAccComputeInstance_with18TbScratchDisk(t *testing.T) {
+	t.Parallel()
+
+	var instance compute.Instance
+	var instanceName = fmt.Sprintf("tf-test-%s", RandString(t, 10))
+
+	VcrTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    TestAccProviders,
```

Comment:
we need to change test provider to `ProtoV5ProviderFactories: ProtoV5ProviderFactories(t),` after https://github.com/GoogleCloudPlatform/magic-modules/pull/7478

Reply:
Changed. Rerunning acceptance test locally now.

Reply:
Done. Test passed.